Build/build.ps1

```diff
@@ -323,63 +329,243 @@ function Update-RequiredModules {
     $moduleManifestUpdatedContent
 }
 
+<#
+    .Synopsis
```

Comment:
Optional: You can reuse the structure of the documentation to be the same as the previous functions in the build, meaning that **.Synopsis** should be below the comment sign. The same goes for all other documentation sections.